app/controllers/pdl.py

```diff
@@ -32,6 +32,8 @@
 COMMOD_STATE_DISTRIBUTION_DATA_JSON = "commodities_state_distribution_data.json"
 COMMOD_SUBPROGRAMS_DATA_JSON = "commodities_subprograms_data.json"
 ARC_PLC_DATA_JSON = "arc_plc_payments_current.json.gz"
+ARC_PLC_CURRENT_OBBBA_DATA_JSON = "arc_plc_payments_current_obbba.json.gz"
```

Comment:
I will make a note here that once the scenario names are finalized, we need to rename these json.gz files to use those names. This would go along with the task to update scenario names. Thanks.